static/app/utils/performance/contexts/pageError.tsx

```diff
@@ -1,49 +1,78 @@
-import {createContext, useContext, useState} from 'react';
+import {createContext, Fragment, useContext, useRef, useState} from 'react';
 import {Theme} from '@emotion/react';
 
 import {Alert} from 'sentry/components/alert';
+import {IconClose} from 'sentry/icons';
 
-export type PageAlert = {
-  message: string;
+export type PageAlertOptions = {
+  message: string | undefined;
   type: keyof Theme['alert'];
+  dismissible?: boolean;
```

Comment:
Looks like other parts of the code use `isDismissible` instead

Reply:
👍